packages/cli/tsconfig.json

```diff
@@ -1,27 +1,14 @@
 {
+	"extends": "../../tsconfig.json",
 	"compilerOptions": {
-		"lib": [
-			"es2017",
-			"ES2020.Promise"
-		],
+		"outDir": "dist",
 		"types": [
 			"node",
 			"jest"
 		],
-		"module": "commonjs",
-		"noImplicitAny": true,
-		"removeComments": true,
-		"strictNullChecks": true,
-		"forceConsistentCasingInFileNames": true,
-		"noImplicitReturns": true,
-		"preserveConstEnums": true,
-		"declaration": true,
-		"outDir": "./dist/",
-		"target": "es2017",
```

Comment:
To support nodejs 14+, `"target": "es2020"` is recommended. But our current frontend tooling isn't handling `workflow` being compiled for `es2020`. Once we upgrade babel (or switch away from it), we can change the `target` across the code to `es2020`, as that'll produce smaller (and likely faster) JS files.